File: tags/NHI1-0.14/theLink/example/php/mulclient.php
<?php
#+
#§  \file       theLink/example/php/mulclient.php
#§  \brief      \$Id$
#§  
#§  (C) 2010 - NHI - #1 - Project - Group
#§  
#§  \version    \$Rev$
#§  \author     EMail: aotto1968 at users.berlios.de
#§  \attention  this software has GPL permissions to copy
#§              please contact AUTHORS for additional information
#§

$srv = new MqS();
try {
  $srv->ConfigSetName("MyMul");
  $srv->LinkCreate($argv);
  $srv->SendSTART();
  $srv->SendD(3.67);
  $srv->SendD(22.3);
  $srv->SendEND_AND_WAIT("MMUL");
  print($srv->ReadD());
} catch (Exception $ex) {
  $srv->ErrorSet($ex);
}
$srv->Exit();

?>

File: tags/NHI1-0.16/theLink/example/csharp/mulclient.cs
/**
 *  \file       theLink/example/csharp/mulclient.cs
 *  \brief      \$Id$
 *  
 *  (C) 2009 - NHI - #1 - Project - Group
 *  
 *  \version    \$Rev$
 *  \author     EMail: aotto1968 at users.berlios.de
 *  \attention  this software has GPL permissions to copy
 *              please contact AUTHORS for additional information
 */
using System;
using csmsgque;
namespace example {
  sealed class mulclient : MqS {
    static void Main(string[] argv) {
      mulclient ctx = new mulclient();
      try {
	ctx.ConfigSetName("MyMul");
	ctx.LinkCreate(argv);
	ctx.SendSTART();
	ctx.SendD(3.67);
	ctx.SendD(22.3);
	ctx.SendEND_AND_WAIT("MMUL", 5);
	Console.WriteLine(ctx.ReadD());
      } catch (Exception ex) {
        ctx.ErrorSet (ex);
      }
      ctx.Exit();
    }
  }
}
